dbt/dbt-run.py

```diff
@@ -27,21 +31,33 @@ def setup():
   with connection() as conn:
       with conn.cursor() as cur:
           cur.execute(f"""
-              CREATE SCHEMA IF NOT EXISTS {os.getenv('POSTGRES_SCHEMA')};
+              CREATE SCHEMA IF NOT EXISTS {SCHEMA_NAME};
           """)
       conn.commit()
 
   with connection() as conn:
       with conn.cursor() as cur:
           cur.execute(f"""
               CREATE TABLE IF NOT EXISTS
-              {os.getenv('POSTGRES_SCHEMA')}._dataemon (
+              {SCHEMA_NAME}._dataemon (
                   inserted_on TIMESTAMP DEFAULT NOW(),
                   packages jsonb, manifest jsonb
               )
           """)
       conn.commit()
 
+  with connection() as conn:
+      with conn.cursor() as cur:
+          cur.execute(f"""
+              CREATE TABLE IF NOT EXISTS
+              {BATCH_STATUS_TABLE} (
+                  id SERIAL PRIMARY KEY,
+                  timestamp TIMESTAMP,
+                  status TEXT
+              )
+          """)
+      conn.commit()
```

Comment:
question: Why does the connection have to be reestablished for every query? Couldn't the same connection be used for a group of queries or the whole process?